src/SFA.DAS.EAS.Portal.Worker/Extensions/MessageContextContextExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NServiceBus;
+using SFA.DAS.EAS.Portal.Application.Services;
+using System;
+using System.Globalization;
+
+namespace SFA.DAS.EAS.Portal.Worker.Extensions
+{
+    public static class MessageContextContextExtensions
```

Comment:
We could call this MessageContextExtensions.

Reply:
Yep!

Reply:
👍